docs/attestations.md

```diff
@@ -0,0 +1,76 @@
+# Attestations
```

Comment:
Do we need a separate "attestations" page. Could we have 3 pages: SBOM, Provenance, Attestations-storage. The First 2 are for users, last is for frontend creators.